.gitignore

```diff
@@ -38,5 +38,8 @@ coverage/
 parsed-docs.json
 packages/**/tsdoc.json
 
+# Compiled story files
+packages/**/stories.js
```

Comment:
I know we're not going to even see these files ourselves, but should these be `<ComponentName>.stories.js` like the rest of our files?